README.md

```diff
@@ -30,64 +30,79 @@ class AuditLogInstrumentationConfig {
 
 GraphQL audit creator can take extra option like
 
-- custom audit log destination data source
-  Please implement ``AuditLogSender``
+### Custom audit log destination data source
+
+You can provide your own implementation of ``AuditLogSender`` so that the logs can, for example, be sent to the database, to the event queue or to a file.
+
+Please implement ``AuditLogSender``
 
 ```
  /**
    * Enable sending audit log;
    *
    * @param auditLogSender {@link AuditLogSender}
-   * @return {@link AuditLogInstrumentationCreator}
+   * @return {@link AuditLogInstrumentationBuilder}
    */
-  public AuditLogInstrumentationCreator withActionLogSender(AuditLogSender auditLogSender) {
+  public AuditLogInstrumentationBuilder withActionLogSender(AuditLogSender auditLogSender) {
     this.auditLogSender = auditLogSender;
     return this;
   }
  ```
 
-- custom user context provider. Please implement interface ``UserProvider`` 
+### Custom user context provider
+
+Please implement interface ``UserProvider`` 
 
 ```
   /**
    * Set user context provider.
    *
    * @param userProvider {@link UserProvider}
-   * @return {@link AuditLogInstrumentationCreator}
+   * @return {@link AuditLogInstrumentationBuilder}
    */
-  public AuditLogInstrumentationCreator withUserProvider(UserProvider userProvider) {
+  public AuditLogInstrumentationBuilder withUserProvider(UserProvider userProvider) {
     this.userProvider = userProvider;
     return this;
   }
 ```
 
-- anonymization can be enabled to not send data which you can easily pick up.
-  Please implement interface ``AnonymizedFieldsSetupProvider``
+### Anonymization
+
+Anonymization can be enabled to not send data which you can easily pick up.
+This function ensures compliance of the log audit with the GDPR.
+Thanks to it, you can, for example, anonymize personal data in an audit log.
+
+Please implement interface ``AnonymizedFieldsSetupProvider``
 
 ```
   /**
    * Enable log anonymizer.
    *
    * @param auditLogAnonymizer {@link AuditLogAnonymizer}
-   * @return {@link AuditLogInstrumentationCreator}
+   * @return {@link AuditLogInstrumentationBuilder}
    */
-  public AuditLogInstrumentationCreator withAuditLogAnonymizer(
+  public AuditLogInstrumentationBuilder withAuditLogAnonymizer(
       AuditLogAnonymizer auditLogAnonymizer) {
     this.auditLogAnonymizer = auditLogAnonymizer;
     return this;
   }
 ```
 
-- extra field like input query field can be logged.
-  Please implement interface ``AdditionalFieldsSetupProvider``
+### Extra field
+
+Extra field like input query field can be logged.
+It can be useful in situations where we want the audit log to include fields even if the client does not ask for them, 
```

Comment:
It can be useful in situations where we want the audit log to include fields even if the client does not ask for them. 
For example, the client **fetches** the user's email, and we want that **user's** ID to be included in each audit log containing user entity regardless **of** the client **fetches** the user ID.

Reply:
ok